autofac-tests/Ping/Providers/HttpPingProvider.cs

```diff
@@ -0,0 +1,15 @@
+namespace autofac_tests
+{
+    using System;
+
+    public class HttpPingProvider : IPingProvider
+    {
+        private static readonly Random rnd = new Random(Environment.TickCount);
```

Comment:
Random rnd - Static readonly is same as constants and usually uses capital letter in name.